### ISSUE TITLE
World Map cant zoom

### ISSUE THREAD

**Issue by Ephaltes** [mikepauer/Carbonite#641]
Hello,
I just wanted to add, that when I have the worldmap the zoom in isnt working properly
when I zoom in at some point it starts to zoom out again, and when I somehow managed to zoom in and am hovering over e.g. Durotar with my mouse it is not showing the zone information like lvl from to, it is showing an area which is next to Durotar.
At last the scale function isnt working for me it is always zoomed out, so that i can see the whole continent

Regards

P.S: WoW 7.2

**Comment by IrcDirk** [comment]
Are u using latest version from github? Did u tried with all addons exempt Carbonite? Please post screenshots or video.

**Comment by fdellwing** [comment]
I know this problem as well, but hard to describe. Will try to give more information these days.

**Comment by IrcDirk** [comment]
I have sometimes problemes with mouse scrolling. While zoomin it zooms out...

**Comment by fdellwing** [comment]
Yes, this and the think with not showing the zone details sometimes from another zone.
